Fix/model statement (#154)
smartmind-team/thanosql-docs: docs/en/how-to_guides/ThanoSQL_model/AutomlRegressor.md

(query_expr)
```

 __OPTIONS Clause__

```sql
OPTIONS(
    (target_col=column_name),
    [features_to_drop=[column_name, ...]],
    [impute_type={'simple'|'iterative'}],
    [datetime_attribs=[column_name, ...]],

>     [outlier_method={'pca'|'iso'|'knn'}],

Reviewer: ```suggestion
    [outlier_method={'knn'|'iso'|'pca'}],
```

    [time_left_for_this_task=VALUE],
    [overwrite={True|False}]
    )
```

The "__OPTIONS__" clause allows you to change the value of a parameter. The definition of each parameter is as follows.

- "target_col": the name of the column containing the target value of the classification model (str, default: 'target') 

> - "features_to_drop": selects columns that cannot be used for training (int, optional, default: 300)

Reviewer: wrong option
```suggestion
- "features_to_drop": selects columns that cannot be used for training (list[str], optional)
```

- "impute_type": determines how empty values ​​(NaNs) are handled (str, optional, 'simple'|'iterative' , default: 'simple') 

> > "simple": for empty values, categorical variables are treated as the most common value and continuous variables are treated as the mean

Reviewer: ```suggestion
> "simple": for empty values, categorical variables are treated as the most common value and continuous variables are treated as the mean. 
```

> > "simple": for empty values, categorical variables are treated as the most common value and continuous variables are treated as the mean

Reviewer: ```suggestion
> "simple": for empty values, categorical variables are treated as the most common value and continuous variables are treated as the mean  
```

> "iterative": applies an algorithm that predicts empty values with the remaining properties

> - "datetime_attribs": selects columns corresponding to the date (str, optional)

Reviewer: ```suggestion
- "datetime_attribs": selects columns corresponding to the date (list[str], optional)
```

> - "outlier_method": determines how outliers are handled in the table (str, optional, default: 'iso')
> > "pca": detect abnormal samples by reducing and restoring dimensions using the Principal Component Analysis (PCA)
> > "iso": use Isolation Forest to randomly branch the data table on a tree basis, isolate all observations, and detect abnormal samples (Works efficiently on datasets with many variables)  
> >  "knn": use a K-NN-based approach to detect abnormal samples based on the distance between each data

Reviewer: ```suggestion
- "outlier_method": determines how outliers are handled in the table (str, optional, 'knn'|'iso'|'pca' default: 'iso')
>  "knn": use a K-NN-based approach to detect abnormal samples based on the distance between each data. 
> "iso": use Isolation Forest to randomly branch the data table on a tree basis, isolate all observations, and detect abnormal samples (Works efficiently on datasets with many variables)  
> "pca": detect abnormal samples by reducing and restoring dimensions using the Principal Component Analysis(PCA)
```

(query_expr)
```

 __OPTIONS Clause__

```sql
OPTIONS(
    (target_col=column_name),
    [features_to_drop=[column_name, ...]],
    [impute_type={'simple'|'iterative'}],
    [datetime_attribs=[column_name, ...]],

>     [outlier_method={'pca'|'iso'|'knn'}],

Reviewer: ```suggestion
    [outlier_method={'knn'|'iso'|'pca'}],
```

    [time_left_for_this_task=VALUE],
    [overwrite={True|False}]
    )
```

The "__OPTIONS__" clause allows you to change the value of a parameter. The definition of each parameter is as follows.

- "target_col": the name of the column containing the target value of the classification model (str, default: 'target') 

> - "features_to_drop": selects columns that cannot be used for training (int, optional, default: 300)

Reviewer: ```suggestion
- "features_to_drop": selects columns that cannot be used for training (list[str], optional)
```

- "impute_type": determines how empty values ​​(NaNs) are handled (str, optional, 'simple'|'iterative' , default: 'simple') 
> "simple": for empty values, categorical variables are treated as the most common value and continuous variables are treated as the mean
> "iterative": applies an algorithm that predicts empty values with the remaining properties

> - "datetime_attribs": selects columns corresponding to the date (str, optional)

Reviewer: ```suggestion
- "datetime_attribs": selects columns corresponding to the date (list[str], optional)
```

> - "outlier_method": determines how outliers are handled in the table (str, optional, default: 'iso')

Reviewer: ```suggestion
- "outlier_method": determines how outliers are handled in the table (str, optional, 'knn'|'iso'|'pca', default: 'knn')
```

> > "pca": detect abnormal samples by reducing and restoring dimensions using the Principal Component Analysis (PCA)
> > "iso": use Isolation Forest to randomly branch the data table on a tree basis, isolate all observations, and detect abnormal samples (Works efficiently on datasets with many variables)  
> >  "knn": use a K-NN-based approach to detect abnormal samples based on the distance between each data

Reviewer: ```suggestion
>  "knn": use a K-NN-based approach to detect abnormal samples based on the distance between each data  
> "iso": use Isolation Forest to randomly branch the data table on a tree basis, isolate all observations, and detect abnormal samples (Works efficiently on datasets with many variables)  
> "pca": detect abnormal samples by reducing and restoring dimensions using the Principal Component Analysis (PCA)
```

- "overwrite": determines whether to overwrite a model if it already exists. If set as True, the old model is replaced with the new model (bool, optional, True|False, default: False)

## __PREDICT Syntax__

Use the "__PREDICT__" statement to apply AI models to perform prediction, classification, recommendation, and more. The "__PREDICT__" statement can preprocess the dataset defined by the query_expr that comes after the "__AS__" clause.

```sql
query_statement:
    query_expr

PREDICT USING (model_name_expression)
OPTIONS (
    expression [ , ...]
    )
AS
(query_expr)
```

__OPTIONS Clause__

```sql
OPTIONS(
    [result_col=column_name],
    [table_name=expression] 
    )
```

The "__OPTIONS__" clause allows you to change the value of a parameter. The definition of each parameter is as follows.

- "result_col": the column that contains the predicted results (str, optional, default: 'predict_result')
- "table_name": the table name to be stored in the ThanoSQL workspace database. If a previously used table is specified, the existing table will be replaced by the new table with a 'predict_result' column (str, optional)

__PREDICT Example__

> An example "__{syntax name}__" query can be found in [Create a Regression Model Using AutoML](/en/tutorials/thanosql_ml/regression/automl_regression/).

Reviewer: ```suggestion
An example "__PREDICT__" query can be found in [Create a Regression Model Using AutoML](/en/tutorials/thanosql_ml/regression/automl_regression/).
```

```sql
%%thanosql
PREDICT USING bike_regression
OPTIONS (
    result_col='predict_result',

>     table_name='bike_test'

Reviewer: ```suggestion
    table_name='bike_sharing_test'
```

    )
AS
SELECT *

> FROM bike_sharing_test

Reviewer: ```suggestion
FROM bike_sharing_test
LIMIT 10
```
